src/luscherZeta.c

```diff
@@ -5,6 +5,8 @@
 
 #include <complex.h>
 
+//#include <Rcpp.h>
```

Comment:
this is not present in the master branch, you can just delete it

Reply:
Now this line is deleted, too.